presto-main/src/main/java/com/facebook/presto/util/MoreMaps.java

```diff
@@ -26,7 +26,12 @@ private MoreMaps() {}
 
     public static <K, V> Map<K, V> mergeMaps(Map<K, V> map1, Map<K, V> map2, BinaryOperator<V> merger)
     {
-        return Stream.of(map1, map2)
+        return mergeMaps(Stream.of(map1, map2), merger);
+    }
+
+    public static <K, V> Map<K, V> mergeMaps(Stream<Map<K, V>> mapStream, BinaryOperator<V> merger)
```

Comment:
in the dependent #7212, in [this line](https://github.com/prestodb/presto/pull/7212/commits/ae283fca46deaa9ab0d91763d6a12b86c6edfec6#diff-943d7ae1ce365f0b04a2a7e5fdf17339R65)